README.md

```diff
@@ -33,8 +32,7 @@ Or if you're using Windows, the following should get you going in a Git Bash she
 mkdir hermes_workingdir
 cd hermes_workingdir
 git -c core.autocrlf=false clone https://github.com/facebook/hermes.git
-hermes/utils/build/build_llvm.py --build-system='Visual Studio 16 2019' --cmake-flags='-A x64' --distribute
-hermes/utils/build/configure.py --build-system='Visual Studio 16 2019' --cmake-flags='-A x64 -DLLVM_ENABLE_LTO=OFF' --distribute
+hermes/utils/build/configure.py --build-system='Visual Studio 16 2019' --cmake-flags='-A x64 --distribute
```

Comment:
You're missing a upper single colon in your commit after the cmake flag(s) x64 in the readme on line 35